src/Field/Configurator/AssociationConfigurator.php

```diff
@@ -26,19 +31,23 @@ final class AssociationConfigurator implements FieldConfiguratorInterface
 {
     private EntityFactory $entityFactory;
     private AdminUrlGenerator $adminUrlGenerator;
+    private RequestStack $requestStack;
+    private ControllerFactory $controllerFactory;
 
-    public function __construct(EntityFactory $entityFactory, AdminUrlGenerator $adminUrlGenerator)
+    public function __construct(EntityFactory $entityFactory, AdminUrlGenerator $adminUrlGenerator, RequestStack $requestStack, ControllerFactory $controllerFactory)
     {
         $this->entityFactory = $entityFactory;
         $this->adminUrlGenerator = $adminUrlGenerator;
+        $this->requestStack = $requestStack;
+        $this->controllerFactory = $controllerFactory;
     }
 
     public function supports(FieldDto $field, EntityDto $entityDto): bool
     {
         return AssociationField::class === $field->getFieldFqcn();
     }
 
-    public function configure(FieldDto $field, EntityDto $entityDto, AdminContext $context): void
+    public function configure(FieldDto $field, EntityDto $entityDto, AdminContext $context, FieldContext $fieldContext = null): void
```

Comment:
```suggestion
    public function configure(FieldDto $field, EntityDto $entityDto, AdminContext $context): void
```

I don't find any use of $fieldContext. Probably something left behind?

Reply:
Thanks! Yes, left behind.